gnuradio-runtime/lib/pmt/pmt.cc

```diff
@@ -290,7 +249,7 @@ string_to_symbol(const std::string &name)
     if (name == _symbol(sym)->name())
       return sym;		// Yes.  Return it
   }
-  
+
```

Comment:
_cough_ whitespace fixing in functional commit _cough_ Frankly, I want this so much that I don't care at this point